src/styles/blocks/main.scss

```diff
@@ -0,0 +1,184 @@
+@use '../utils/mixins.scss' as *;
+
+.main {
+  padding: 0 20px;
+  background-color: #1D1D1D;
+
+  @include grid;
+
+  &__paragraph {
+    font-family: Poppins, sans-serif;
+    font-size: 23px;
+    font-weight: 500;
+    line-height: 32.2px;
+    text-align: left;
+    color: #FFF;
+    grid-column: 1 / 3;
+    margin-bottom: 92px;
+
+    @include for-tablet {
+      grid-column: 4 / 7;
+      margin-top: 120px;
+      margin-bottom: 132px;
+    }
+
+    @include for-desktop {
+      grid-column: 6 / 12;
+      margin-top: 120px;
+      margin-bottom: 164px;
+    }
+  }
+
+  &__title {
+    margin-bottom: 32px;
+    grid-column: 1 / 3;
+
+    @include for-tablet {
+      grid-column: 1 / 4;
+      margin-top: 120px;
+    }
+
+    @include for-desktop {
+      grid-column: 1 / 5  ;
+      margin-top: 120px;
```

Comment:
I'd create a separate container for this title && paragraph block to ease the positioning.